src/app/search/search-documents-table-rows/search-document-table-rows.component.ts

```diff
@@ -0,0 +1,105 @@
+import { Component, Input, Output, OnInit, OnDestroy, EventEmitter, ChangeDetectorRef } from '@angular/core';
+import { Subject } from 'rxjs';
+import { TableComponent } from 'app/shared/components/table-template/table.component';
+import { TableObject } from 'app/shared/components/table-template/table-object';
+import { Router, ActivatedRoute } from '@angular/router';
+import { ApiService } from 'app/services/api';
+import { Utils } from 'app/shared/utils/utils';
+
+@Component({
+  selector: 'tbody[app-document-table-rows]',
+  templateUrl: './search-document-table-rows.component.html',
+  styleUrls: ['./search-document-table-rows.component.scss']
+})
+
+export class DocSearchTableRowsComponent implements OnInit, OnDestroy, TableComponent {
+  private ngUnsubscribe: Subject<boolean> = new Subject<boolean>();
+  @Input() data: TableObject;
+  @Output() selectedCount: EventEmitter<any> = new EventEmitter();
+
+  public documents: any;
+  public paginationData: any;
+  public showFeatured = true;
+  private lists: any[] = [];
+
+  public currentUrl: String = '';
+
+  constructor(
+    private _changeDetectionRef: ChangeDetectorRef,
+    private route: ActivatedRoute,
+    private api: ApiService,
+    private router: Router,
+    private utils: Utils
+  ) {
+    let currRoute = router.url.split(';')[0];
```

Comment:
The current route will actually be split up nicely for you in the `ActivatedRoute` component. This remove the need to manually split and look for the current route. 

Something like: `this.currentUrl = route.url[0].path` (just guessing, would need to check)

Reply:
the currRoute and url weren't used, removing these vars